packages/contracts/contracts/interfaces/v0.8.x/IAdminACLV0.sol

```diff
@@ -23,6 +23,18 @@ interface IAdminACLV0 {
     /// super admin address
     function superAdmin() external view returns (address);
 
+    /**
+     * @notice Allows superAdmin change the superAdmin address.
+     * @param _newSuperAdmin The new superAdmin address.
+     * @param _genArt721CoreAddressesToUpdate Array of genArt721Core
+     * addresses to update to the new superAdmin, for indexing purposes only.
+     * @dev this function is gated to only superAdmin address.
+     */
+    function changeSuperAdmin(
+        address _newSuperAdmin,
+        address[] calldata _genArt721CoreAddressesToUpdate
+    ) external;
+
```

Comment:
@lindsgil I think this needs to be broken out into a new interface, because I believe it will prevent _current_ AdminACLV0's from transferring ownership to the new AdminACLV0's, because when switching there is a forward-looking check for conformance on the new AdminACL for `type(IAdminACLV0).interfaceId`, which changes if we change this interface file.

Reply:
let's also add a test that hard-codes that this file's interfaceId never changes 😅 because how would anyone ever know this 😅 😅

Reply:
Thanks for catching this!! 😅 😅 I added tests to check that AdminACLV0 supports hardcoded IAdminACLV0 and IAdminACLV0_Engine (new interface) interface IDs.. LMK what you think of that approach or if there's a more robust way!

```
    it("supports IAdminACLV0 and IAdminACLV0_Engine Interface IDs", async function () {
      const config = await loadFixture(_beforeEach);
      // @dev these interface IDs are hardcoded here to ensure
      // any future changes do not break deployed contracts
      // IAdminACLV0 interface ID
      const iadminACLV0InterfaceId = 0xc00707bc;
      // IAdminACLV0_Engine interface ID
      const iadminACLV0_EngineInterfaceId = 0x377a7d1a;

      expect(
        await config.adminACL_InterfaceBroadcast.supportsInterface(
          iadminACLV0InterfaceId
        )
      ).to.be.true;
      expect(
        await config.adminACL_InterfaceBroadcast.supportsInterface(
          iadminACLV0_EngineInterfaceId
        )
      ).to.be.true;
    });
    ```

Reply:
lgtm as discussed and updated 👍